app.py

```diff
@@ -129,12 +128,44 @@ def _register_indexes(graph: GraphDatabase, index_base_url: str, dry_run: bool =
     return index_urls
 
 
-def _do_schedule_core_solver_jobs(
-    openshift: OpenShift,
-    index_urls: List[str],
-    package_name: str,
-    package_version: str,
-    output: str,
+def _take_data_science_packages() -> List[str]:
+    """Take list of Python Packages for data science."""
+    current_path = Path.cwd()
+    complete_file_path = current_path.joinpath("hundredsDatasciencePackages.yaml")
+
+    with open(complete_file_path) as yaml_file:
+        requested_packages = yaml.safe_load(yaml_file)
+
+    data_science_packages = []
+    for package_name in requested_packages["hundreds_datascience_packages"]:
+        _LOGGER.info("Registering package %r", package_name)
+        data_science_packages.append(package_name)
+
+    return data_science_packages
+
+
+def _schedule_default_packages_solver_jobs(packages: List[str], index_urls: List[str]) -> None:
+    """Run solver jobs for Python packages list selected."""
+    openshift = OpenShift(kubernetes_verify_tls=False)
+
+    for index_url in index_urls:
+        _LOGGER.debug("consider index %r", index_url)
+        source = Source(index_url)
+
+        for package_name in packages:
+            _LOGGER.debug("Obtaining %r versions", package_name)
+
+            try:
+                versions = source.get_package_versions(package_name)
+                for version in versions:
+                    _LOGGER.info("Scheduling package_name %r in package_version %r", package_name, version)
+                    _do_schedule_solver_jobs(openshift, index_urls, package_name, version, _SOLVER_OUTPUT)
+            except Exception as e:
+                _LOGGER.warning(e)
```

Comment:
Directly log an exception and continue? Warning log level will not be propagated to monitoring services by default.

```
_LOGGER.exception(str(exc))
```